pkg/asset/agent/manifests/agentclusterinstall.go

```diff
@@ -38,6 +39,38 @@ type AgentClusterInstall struct {
 	Config *hiveext.AgentClusterInstall
 }
 
+type agentClusterInstallOnPremPlatform struct {
+	// APIVIPs contains the VIP(s) to use for internal API communication. In
+	// dual stack clusters it contains an IPv4 and IPv6 address, otherwise only
+	// one VIP
+	APIVIPs []string `json:"apiVIPs,omitempty"`
+
+	// IngressVIPs contains the VIP(s) to use for ingress traffic. In dual stack
+	// clusters it contains an IPv4 and IPv6 address, otherwise only one VIP
+	IngressVIPs []string `json:"ingressVIPs,omitempty"`
+}
+
+type agentClusterInstallPlatform struct {
+	// BareMetal is the configuration used when installing on bare metal.
```

Comment:
Nit: `baremetal`

Reply:
This is copied from installer types package I think

Reply:
ok, then